jigna/js/build.js

```diff
@@ -68,3 +69,37 @@ concatenate({
         'jigna'
     ]
 });
+
+// jigna-vue.js
+concatenate({
```

Comment:
`jigna-angular.js` does this: `jigna.angular.app.run`, so I assume that means angular is running.